### PR TITLE
FUL-503: - added go source for fetching and storing area picking deta…

### PR DESCRIPTION
**Pull request by rene-lichtenberg**
…il informations for grafana

- .gitignore adhusted
- init.sql added with some example data (for start in MYSQL format)

## Description

Solves JIRA_ID
FUL-503

## Proposed changes

- 
- 
- 

## Checklist

- [ ] Your teams Definition of Done and / or things you like to remind yourself of go here!
- [ ] As an inspiration, find example items below
- [ ] I have performed a self-review of my own code.
- [ ] I have added tests that prove my changes are working.
- [ ] Any dependent changes have been merged and published.

### CODE REVIEW

File: infrastructure/rds.tf [hunk 1]
```diff
@@ -1,8 +1,63 @@
...
+}
+output "rds_password" {
+  value = module.rds_module.rds_password_value
+}
+output "rds_host" {
+  value = module.rds_module.rds_address
+}
+output "rds_port" {
+  value = module.rds_module.rds_port
+}
```
Comment (PhEigl) [inline infrastructure/rds.tf:63]: are you outputting these to see it in the console? I think we don't need those

Comment (JulianGogK24) [reply infrastructure/rds.tf:63]: for debugging and connecting via dbeaver I would like to have them until we merge to master. What do you say?

File: infrastructure/rds.tf [hunk 2]
```diff
@@ -1,8 +1,63 @@
...
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "chmod +x /tmp/psql-init.sh",
+      "bash /tmp/psql-init.sh",
+      "rm -rf /tmp/psql-*"
+    ]
+  }
+}
```
Comment (PhEigl) [inline infrastructure/rds.tf:50]: NICE!

File: infrastructure/rds.tf [hunk 3]
```diff
@@ -1,8 +1,63 @@
...
+
+resource "null_resource" "rds_db_setup" {
+  # runs after database and security group providing external access is created
+  depends_on = ["module.rds_module", "module.rds-security-group", "module.lambda"]
+  triggers = {
+    always_run = "${timestamp()}"
+  }
+  connection {
+    type        = "ssh"
+    host        = "bastion-host.dev.kfzteile24.io"
```
Comment (PhEigl) [inline infrastructure/rds.tf:27]:
```suggestion
    host        = "bastion-host.${var.environment}.kfzteile24.io"
```

Comment (JulianGogK24) [reply infrastructure/rds.tf:27]: nice catch! Thx

Comment (dkyborg) [reply infrastructure/rds.tf:27]: I would even go one step further and get the name directly from the output of the bastion module. I will take care of the output (its currently not available) and inform you then.

Comment (JulianGogK24) [reply infrastructure/rds.tf]: added Philipps approach.
Made a ticket to change to your Idea @dkyborg

File: infrastructure/rds.tf (hunk 1 above)
Comment (JulianGogK24) [reply infrastructure/rds.tf]: removed all sensitive data

File: src/models/database.go [hunk 4]
```diff
@@ -0,0 +1,580 @@
...
+            ConsolidationAreaDetails.PriorityColor = 1
+        } else {
+            ConsolidationAreaDetails.PriorityColor = 0
+        }
+        consolidationAreaDetails = append(consolidationAreaDetails, ConsolidationAreaDetails)
+    }
+    
+    query := `
+            SELECT STRAIGHT_JOIN
+                IF(apl_pb='LUD', 'LUD', IF(apl_ar LIKE 'P11%', 'BT11', IF(apl_ar LIKE 'P3%', 'BT20', 'BT20'))) AS 'Dashboard',
```
Comment (maciej-kups-k24) [inline src/models/database.go:481]: This seems to have no effect: 
`IF(apl_ar LIKE 'P3%', 'BT20', 'BT20')`
The result will allways be BT20. Is this on purpose or by mistake?

Comment (rene-lichtenberg) [reply src/models/database.go:481]: Well seen!
Actually, only the areas that start with P3 belong to BT20.
It is based on a short-term decision to also display all remaining areas in the BT20 overview.
So I would leave it that way.

**Pull request merged**